src/unix.rs

```diff
@@ -0,0 +1,8 @@
+
+/// Unix specific extension methods.
```

Comment:
I just meant copying the docs in the standard `DirEntryExt`, rather than linking to that trait. So our `walkdir::unix::DirEntryExt` trait looks like:

```rust
/// Unix-specific extension methods for `walkdir::DirEntry`
pub trait DirEntryExt {
    /// Returns the underlying `d_ino` field in the contained `dirent` structure.
    fn ino(&self) -> u64;
}
```